9.0/9.0.1 Commerce/download-and-extract.ps1

```diff
@@ -0,0 +1,206 @@
+    $commercePackageDestination = $([io.path]::combine($DownloadFolder, $commercePackageFileName)).ToString()
+    
+    Write-Host "Saving $CommercePackageUrl to $commercePackageDestination - if required" -ForegroundColor Green
+    if (!(Test-Path $commercePackageDestination)) {
+        Invoke-SitecoreDownload $CommercePackageUrl $commercePackageDestination
+    }
+    
+
+    $sxaPackageUrl = "https://dev.sitecore.net/~/media/573443081B494E2B9D83D3208B549E49.ashx"
+    $sxaPackageDestination = $([io.path]::combine($CommerceAssetFolder, "Sitecore Experience Accelerator 1.7 rev. 180410 for 9.0.zip"))
+    if (!(Test-Path $sxaPackageDestination)) {
+        Invoke-SitecoreDownload $sxaPackageUrl $sxaPackageDestination
+    }
+   
+    $spePackageUrl = "https://marketplace.sitecore.net/services/~/download/3D2CADDAB4A34CEFB1CFD3DD86D198D5.ashx?data=Sitecore%20PowerShell%20Extensions-4.7.2%20for%20Sitecore%208&itemId=6aaea046-83af-4ef1-ab91-87f5f9c1aa57"
+    $spePackageDestination = $([io.path]::combine($CommerceAssetFolder, "Sitecore PowerShell Extensions-4.7.2 for Sitecore 8.zip"))
+   
+    if (!(Test-Path $spePackageDestination)) {
+        Invoke-SitecoreDownload $spePackageUrl $spePackageDestination
+    }
+
+    $msbuildNuGetUrl = "https://www.nuget.org/api/v2/package/MSBuild.Microsoft.VisualStudio.Web.targets/14.0.0.3"
+    $msbuildNuGetPackageFileName = "msbuild.microsoft.visualstudio.web.targets.14.0.0.3.nupkg"
+    $msbuildNuGetPackageDestination = $([io.path]::combine($DownloadFolder, $msbuildNuGetPackageFileName))
+    
+    Write-Host "Saving $msbuildNuGetUrl to $msbuildNuGetPackageDestination - if required" -ForegroundColor Green
+    if (!(Test-Path $msbuildNuGetPackageDestination)) {
+        Invoke-WebRequest  -Uri $msbuildNuGetUrl -OutFile $msbuildNuGetPackageDestination
+        #Start-BitsTransfer -source $msbuildNuGetUrl -Destination $msbuildNuGetPackageDestination
+    }
+    $aspnetCoreGetUrl = "https://aka.ms/dotnetcore-2-windowshosting"
+    $aspnetCoreFileName = "DotNetCore.2.0.5-WindowsHosting.exe"
+    $aspnetPackageDestination = $([io.path]::combine($DownloadFolder, $aspnetCoreFileName))
+    
+    Write-Host "Saving $aspnetCoreGetUrl to $aspnetPackageDestination - if required" -ForegroundColor Green
+    if (!(Test-Path $aspnetPackageDestination)) {
+        Start-BitsTransfer -source $aspnetCoreGetUrl -Destination $aspnetPackageDestination
+    }
+    
+    $netCoreSDKUrl = "https://download.microsoft.com/download/0/F/D/0FD852A4-7EA1-4E2A-983A-0484AC19B92C/dotnet-sdk-2.0.0-win-x64.exe"
+    $netCoreSDKFileName = "dotnet-sdk-2.0.0-win-x64.exe"
+    $netCoreSDKPackageDestination = $([io.path]::combine($DownloadFolder, $netCoreSDKFileName))
+    
+    Write-Host "Saving $netCoreSDKUrl to $netCoreSDKPackageDestination - if required" -ForegroundColor Green
+    if (!(Test-Path $netCoreSDKPackageDestination)) {
+        Start-BitsTransfer -source $netCoreSDKUrl -Destination $netCoreSDKPackageDestination
+    }
```

Comment:
Why not take the latest version of 2.1.0? And I think we need to explicitly add something to the readme file that these installers (pre-reqs) need to be installed first before running the deploy script

Reply:
won't it be installed once you have XP 9 on the machine?

Reply:
XP 9 doesn't require .NET core, that's why it was part of my XC 9 install.